src/test/java/pl/edu/mimuw/logic/SolverTest.java

```diff
@@ -5,7 +5,20 @@
 class SolverTest {
 
   @Test
-  void testFormulaButChangeThisFunctionName() {
+  void testHamlet() {
+    Formula hamlet = FormulaFactory.or(new Var("to_be"), FormulaFactory.neg(new Var("to_be")));
+    Valuation tempV = Solver.solve(hamlet);
+    assert(hamlet.eval(tempV));
```

Comment:
testy nieprawidłowo sprawdzają oczekiwane wartości, nie robiliśmy tego przez `assert` a  np . `assertEquals`.
To czego użyłeś jest częścią Javy i nie jest wlaczone domyslnie, nam zalezy na uzyciu bibliotegi do testowania i jej funkcji